### DISCUSSION TITLE
InputField with TextMeshPro

### DISCUSSION THREAD

**Discussion by slimshader** [kyubuns/AkyuiUnity#28] [Q&A] [answered]
Hi,

I am trying to create a form for username etc. but unable to import TMP_InputField. I have TMP Parser and Triggers configured and Text components are imported fine, only problem is with InputField.

**Comment by kyubuns** [comment, marked as answer]
Hello!
As it turns out, TMP_InputField is supported and should be usable.
An InputField must be named for a Group, like a Button.
(It's not good that I didn't include an InputField in my sample.)
<img width="142" alt="Screen Shot 2021-09-21 at 21 10 59" src="https://user-images.githubusercontent.com/961165/134168034-d6132f9c-c9aa-4ec7-a603-5d138b3a6976.png">

If this doesn't work, it would be great if you could give me an XD file that reproduces it.

**Reply by slimshader** [reply]
Didn't realize that it had to be group. Works like a charm now, thanks.
